docs/build.gradle.kts

```diff
@@ -14,11 +16,31 @@ dependencies {
     implementation(libs.opentelemetry.exporter.logging)
 }
 
+ktlint {
+    filter {
+        exclude { it.file.path.contains("example-") }
+    }
+}
```

Comment:
why are linter settings updated in docs/build.gradle.kts and not root/build.gradle.kts?